lib/config/load.js

```diff
@@ -165,7 +165,7 @@ function loadFile(options, config, dir, ready) {
     return callback({});
   }
 
-  var filename = path.join(dir, 'nodemon.json');
+  var filename = settings.config || path.join(dir, 'nodemon.json');
```

Comment:
Maybe so. I was mostly just filing this to see if you'd be interested and get some direction. Would you like an actual PR for this?